api/helpers/gamestate/moves/resolve/execute.js

```diff
@@ -33,6 +33,7 @@ module.exports = {
     const { oneOff } = result;
 
     // Move oneOff + twos into scrap and increment turn
+    const fizzles = result.twos.length % 2 === 1;
```

Comment:
We were previously checking whether the one-off fizzles _after_ setting the twos to an empty array, so it never fizzled. Need to check beforehand like so